src/net/sourceforge/kolmafia/request/HermitRequest.java

```diff
@@ -458,7 +458,7 @@ public static final int getAcquirableWorthlessItemCount()
 		if ( InventoryManager.canUseNPCStores() )
 		{
 			int cost = SewerRequest.currentWorthlessItemCost();
-			count += KoLCharacter.getAvailableMeat() / cost;
+			count += ((int)KoLCharacter.getAvailableMeat()) / cost;
```

Comment:
Meat is stored as long but there are many places where it is truncated to an int.  Making count a long would effect several other pieces of code.

Reply:
Will this actually affect the narrowing warning? It's still going to narrowing a long to an int and possibly overflow if out of the int range.

Reply:
I read the warning as being about an implicit conversion.  An explicit cast may be a bad thing but compilers and other tools can deal with explicit casts differently than implicit casts.  The explicit cast may be wrong but it was a deliberate.  That said, I don't know how to run the security check locally so I don't know for sure.  If this is submitted and the warning still occurs after test then I just make one more iteration and this time select the wider ranging changes.

Reply:
Missed one but the cast seems to have addressed the warning for the ones I did catch.